src/Auth/Auth.js

```diff
@@ -86,13 +75,24 @@ export default class Auth {
     // Remove isLoggedIn flag from localStorage
     localStorage.removeItem('isLoggedIn');
 
-    this.auth0.logout();
+    this.logoutForReal();
+    
   }
 
-  isAuthenticated() {
+  isAuthenticated = () => {
     // Check whether the current time is past the
     // access token's expiry time
     let expiresAt = this.expiresAt;
     return new Date().getTime() < expiresAt;
   }
-}
\ No newline at end of file
+
+  logoutForReal = () => {
```

Comment:
It might be nice to think of a more descriptive name for this and the logout function :/